src/components/add-review-screen/add-review-screen.jsx

```diff
@@ -74,8 +74,9 @@ AddReviewScreen.propTypes = {
 
 const mapStateToProps = (state, ownProps) => {
   const {id} = ownProps;
+  const idInteger = parseInt(id, 10);
```

Comment:
Лучше использовать объект Number

```suggestion
  const idInteger = Number.parseInt(id, 10);
```